scripts/lib/install-node

```diff
@@ -23,6 +23,14 @@ fi
 if [ "$current_node_version" != "v$node_version" ]; then
     export NVM_DIR=/usr/local/nvm
     if ! [ -e "$NVM_DIR/nvm.sh" ]; then
+        # CircleCI forces GitHub urls to use ssh instead of
+        # https by creating a .gitconfig file which which breaks
+        # the git clone in the nvm install script.
```

Comment:
It didn't work out. I also tried running the dockerimage locally. There was no `gitconfig` file in that. I will try ti figure out the origin of this.

Reply:
Hmm, not sure. Do you have a log that shows the failure without this fix?

I'd guess that somehow when that command runs we have something in the environment that causes `ssh` not to find the `.ssh/id_rsa` file. Given an SSH error, you could find out more by inserting something like `ssh -vvv git@github.com` just before (basically, instead of) the command that fails.

Regardless of the exact cause of the issue, it'd be OK to merge this just crudely deleting the file. But
* we should do it right up front, like inside the "install dependencies" block in `.circleci/config.yml`
* we should include in the commit message at least a paste of the key lines of the error we get without it, to help explain why it's there.